rules/windows/defense_evasion_unusual_system_vp_child_program.toml

```diff
@@ -0,0 +1,39 @@
+[metadata]
+creation_date = "2020/08/19"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/19"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies Suspicious child program of the Windows virtual System process. This could indicate code injection a system."
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Unusual System Virtual Process Child Program"
+risk_score = 73
+rule_id = "de9bd7e0-49e9-4e92-a64d-53ade2e66af1"
+severity = "high"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:process and event.type:(start or process_started) and
+  process.parent.pid:4 and 
+  not process.executable:("Registry" or "MemCompression" or "C:\\Windows\\System32\\smss.exe")
```

Comment:
you shouldn't need the extra backslashes, it could just be `C:\Windows\System32\smss.exe`

will `Registry` or `MemCompression` ever evaluate as true since `process.executable` is looking for a path?

Reply:
works also with backslashes, yes both are returned as valid FPs 
![image](https://user-images.githubusercontent.com/64742097/90689543-88597500-e270-11ea-9bad-3569e8486513.png)